docs/source/intervention_models/crm_mgmt/vivarium_lifestyle.rst

```diff
@@ -1,23 +1,69 @@
 Vivarium Modeling Strategy - Lifestyle Intervention
 ---------------------------------------------------
 
-.. todo::
+Overview
+++++++++
+
+Participation in a structured program composed of education sessions and individualized coaching with the goal of reducing body weight and increasing physical activity for simulants. 
+
+
+Scenarios
++++++++++
+In the baseline scenario, no one is enrolled in structured lifestyle modification interventions.
+
+In the alternate scenarios, simulants are eligible for enrollment if their BMI was >= 25, they had no known DM2, and their FPG was 100-125mg/dl (or HbA1C 5.7-6.4). All simulants have a propensity score between 0 to 1 (uniform distribution). If they are eligible and their score is <0.5, the simulant is enrolled. 
+
+Assumptions: There are simulant and/or provider attributes other than eligibility criteria that might affect the likelihood of enrollment/participation but we are not modeling them at this time. 
 
-  Add an overview of the Vivarium modeling section.
+Affected outcomes
++++++++++++++++++
+**Body mass index (weight)**: participants saw a reduction of 6.5 ± 4.7 kg or 6.9 ± 4.5% of initial body weight over the first six months/16 visits. At 3 years after randomization, participants saw a reduction of 4.5 ± 7.6 kg or 4.9 ± 7.4% of initial body weight. 10 years after randomization, participants weighed 2 ± 9 kg on average less than their starting weight.
+
+.. image:: ddpos_weight.jpg
 
 .. todo::
-  Address comment: see my comments on the vivarium_polypill.rst, I expect this doc to hit similar points relevant to lifestyle intervention.
+  Research team: Request data from the DDPOS investigators
 
-Decision tree and algorithm for lifestyle intervention
+.. todo::
+  Software engineers: The research team would prefer to use absolute change in weight, as it allows us to use the results from the meta-analysis described below to account for the decrease in SBP that occurs with weight reduction. This will require the software engineers to assign height values to all simulants as the inputs from GBD are BMI values.
```

Comment:
Do we need to use a distribution of heights that research will provide?  Or will it be acceptable to assume everyone is precisely (for example) 2 meters tall.

Reply:
Will provide distribution of heights/weights/BMI values from NHANEs

Reply:
format for data on height:
age sex year mean (height) std (height)

Could also look at the weighted ensemble distribution, but height should be mostly normally distributed, so okay to just assume that.